nifi-nar-bundles/nifi-hbase-bundle/nifi-hbase-processors/src/main/java/org/apache/nifi/hbase/PutHBaseJSON.java

```diff
@@ -89,6 +89,25 @@
             .defaultValue(COMPLEX_FIELD_TEXT.getValue())
             .build();
 
+    protected static final String STRING_ENCODING_VALUE = "String";
+    protected static final String BYTES_ENCODING_VALUE = "Bytes";
+
+    protected static final AllowableValue FIELD_ENCODING_STRING = new AllowableValue(STRING_ENCODING_VALUE, STRING_ENCODING_VALUE,
+            "Stores the value of each field as a UTF-8 String.");
```

Comment:
That works for me.

That's fair, if questions come in to the mailing list asking to clarify then we can address the description.